tests/Contrib/Unit/JaegerSpanConverterTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Contrib\Unit;
+
+use OpenTelemetry\API\Trace\StatusCode;
+use OpenTelemetry\Contrib\Jaeger\SpanConverter;
+use OpenTelemetry\SDK\Trace\StatusData;
+use OpenTelemetry\Tests\SDK\Util\SpanData;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @coversDefaultClass SpanConverter
```

Comment:
a good starting point would be to just use `@covers` here, meaning coverage will only be generated for the target class (without you having to annotate every method)

Reply:
Think I incidentally did this following the warnings when trying to make the build pass